src/util/mod.rs

```diff
@@ -153,16 +153,39 @@ pub struct HexDisplay<'a> {
     data: &'a [u8],
 }
 
+
+
 impl<'a> Display for HexDisplay<'a> {
     fn fmt(&self, f: &mut Formatter) -> fmt::Result {
-        try!(write!(f, "0x"));
-        for &b in self.data {
-            try!(write!(f, "{:02X}", b));
-        }
+        let mut buf = String::new();
+        print_bytes_to(self.data, &mut buf);
+        write!(f, "{}", buf).unwrap();
```

Comment:
It's not printed as pure hex, it's printed as the same format as protobuf. So we the print format could be human readable.